src/app/adf-home/df-download-page/df-download-page.component.ts

```diff
@@ -1,11 +1,22 @@
 import { Component } from '@angular/core';
+import { DfBreakpointService } from 'src/app/core/services/df-breakpoint.service';
 
 @Component({
   selector: 'df-download-page',
   templateUrl: './df-download-page.component.html',
   styleUrls: ['./df-download-page.component.scss'],
 })
 export class DfDownloadPageComponent {
+  isXSmall: boolean;
+
+  constructor(private breakpointService: DfBreakpointService) {}
+
+  ngOnInit(): void {
+    this.breakpointService.isXSmallScreen.subscribe((isXSmall: boolean) => {
+      this.isXSmall = isXSmall;
+    });
+  }
+
```

Comment:
no need for subscription as teardown require unsubscribe as well

Reply:
Good point!